smlxl/evm.codes
# Bump ethereumjs libraries

Short summary of the changes
- Needed to add two webpack / babel loaders to be able to import `rustbn` used by Ethereumjs.
- There was some minor changes to the types used by EthereumJS so updated to reflect the new types.
     - Mainly `HardforkConfig` -> `HardforkTransitionConfig`
     - `Buffer` -> `Uint8Array`
     - `getActiveOpcodes` isn't exposed on `vm` anymore, just reconstructing the input to EVM was easiest. 

### Test plan
1. Make sure the fork selector correctly reflect the active opcodes (i.e `PUSH0` should not be shown in Paris hardfork) ✅ 
<img width="1279" alt="Screenshot 2024-03-14 at 14 43 32" src="https://github.com/smlxl/evm.codes/assets/5640782/99438936-3018-499a-b950-d2aae3af5889">

2. Playground should have it's storage and stack be correctly updated when running through a contract that interacts with storage ✅ 

<img width="640" alt="Screenshot 2024-03-14 at 14 46 33" src="https://github.com/smlxl/evm.codes/assets/5640782/3ab05939-9229-4413-8db4-fbb601769757">

3. Check that dynamic gas cost is correctly reflected based on version (i.e check that the dynamic cost of `CREATE` is correctly shown in `Shanghai` and is different from the `Paris` version) ✅
